.github/workflows/linux.yaml

```diff
@@ -50,6 +50,8 @@ jobs:
           # Add external PPA, latest version of QT is 5.12.x for Ubuntu 20.04
           sudo add-apt-repository ppa:beineri/opt-qt-5.15.2-focal -y
           sudo apt-get install debhelper devscripts equivs -y
+          pip3 install pyhumps
```

Comment:
I think it would make more sense to have a requirements.txt, and install from that? That way, you only update dependencies once, and you can also set versions to avoid sudden breakages in CI.

Also, I think at this point you can just use `pip` and `python,` no need for the 3 suffix?

Reply:
OK! Follow up for this.